components/Snackbar/src/MDCSnackbarMessageView.m

```diff
@@ -725,9 +739,25 @@ - (CGSize)intrinsicContentSize {
   // Make sure that the height of the image and text is larger than the minimum height;
   height = MAX(kMinimumHeight, height);
 
+  height = MAX(kMinimumHeight, height) + [self contentSafeBottomInset];
   return CGSizeMake(UIViewNoIntrinsicMetric, height);
 }
 
+- (CGFloat)contentSafeBottomInset {
+  // If a bottom offset has been set to raise the HUD, e.g. above a tab bar, we should ignore
+  // any safeAreaInsets, since it is no longer 'anchored' to the bottom of the screen. This is set
+  // by the GOOHUDOverlayView whenever the bottomOffset is non-zero.
```

Comment:
Update the reference to GOOHUDOverlayView

Reply:
Fixed.